### DISCUSSION TITLE
Https everywhere

### DISCUSSION THREAD

**Discussion by rsssxd** [bromite/bromite#1938] [Q&A]
Does the Bromite Https upgrade option also encrypt 3rd party connections like Https Anywhere?

**Comment by Disqu3-mirOir-qt** [comment]
![Screenshot_20220329-135838.jpg](https://user-images.githubusercontent.com/41585567/160606394-9e2d6f8e-ec72-43ac-884b-efb96256d65b.jpg)


always use https , similar https everywhere
